src/User.js

```diff
@@ -1,36 +1,17 @@
+import Client  from './Client.js'
 import Agent from './Agent.js'
-import Client from './Client.js'
 
 class User {
   constructor() {
     this.name = 'Guest'
```

Comment:
This must be the anonymous user that is just viewing destinations upon logging in. Nice idea.